gcloud-java-datastore/src/main/java/com/google/gcloud/datastore/testing/LocalGcdHelper.java

```diff
@@ -396,9 +438,8 @@ private void startGcd(Path executablePath) throws IOException, InterruptedExcept
         .command(gcdAbsolutePath.toString(), "start", "--testing", "--allow_remote_shutdown",
             "--port=" + Integer.toString(port), projectId)
         .directory(gcdPath)
-        .redirectErrorStream()
         .start();
-    processReader = ProcessStreamReader.start(startProcess, "Dev App Server is now running");
+    processReader = ProcessStreamReader.start(startProcess, "Dev App Server is now running", true);
```

Comment:
lets create a separate stream readers and pass the stream (out or err) to each one of them.